README.md

```diff
@@ -14,10 +14,10 @@ None.
 
 ## Role Variables ##
 
-- `logging` - The logging state for ufw.  See options
-  [here](https://docs.ansible.com/ansible/latest/modules/ufw_module.html#parameter-logging).
-- `state` - The state of ufw.  See options
-  [here](https://docs.ansible.com/ansible/latest/modules/ufw_module.html#parameter-state).
+| Variable | Description | Default | Required |
+|----------|-------------|---------|----------|
+| logging | The logging state for ufw.  See options [here](https://docs.ansible.com/ansible/latest/modules/ufw_module.html#parameter-logging).  Quotes are needed around this value because the words `on` and `off` denote boolean values in Ansible. | `"on"` | No |
```

Comment:
Why is "on" in quotes here?  The text `enabled` is not in the next row, and I don't think the quotes agree with what we have been doing in the role variable tables in other Ansible role repos.

Reply:
From the description and carried over from [this comment](https://github.com/cisagov/ansible-role-ufw/blob/e37c9d308783df20d05e2e63f4a3e118fd6410aa/defaults/main.yml#L7-L9):
```
Quotes are needed around this value because the words `on` and `off` denote boolean values in Ansible.
```

Reply:
Doh!  Thank you.